swagger.yaml

```diff
@@ -161,6 +161,39 @@ paths:
       description: Get Tx List
       summary: Get Tx List
 
+  /slashing/validators/{validatorPubKey}/signing_info:
+    get:
+      parameters:
+        - in: path
+          name: validatorPubKey
+          description: validator public key
+          required: true
+          type: string
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: Success
+          schema:
+            $ref: '#/definitions/SigningInfo'
```

Comment:
wrong schema, #/definitions/SigningInfo should be in "result" field.
like
```
        200:
          description: OK
          schema:
            type: object
            properties:
              height:
                type: string
              result:
                type: object
                $ref: '#/definitions/SigningInfo'
```

in our swagger.yaml two schemes are mixed. The first - lcd schema {"height":"123","result":"some data"}. The second - fcd, for /v1 prefixed requests - object

with current schema monitor does not work de facto

Reply:
Great job! ; )